### PR TITLE
refactor pending rides

### PR DESCRIPTION
**Pull request by istzen**
as we discussed in our last Meeting i moved the pending rides from the driveChatPage to the driveDetailView.

### CODE REVIEW

File: lib/drives/pages/drive_detail_page.dart [hunk 1]
```diff
@@ -353,6 +371,25 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
     await _drive?.cancel();
     setState(() {});
   }
+
+  int? _getMessageCount(Drive drive) {
```
Comment (fynsta) [inline lib/drives/pages/drive_detail_page.dart:375]: how would that be null? that makes no sense to me... If there are no messages, then it should be zero.

File: lib/drives/pages/drive_detail_page.dart [hunk 2]
```diff
@@ -163,6 +164,23 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
...
+          Align(
+            alignment: Alignment.centerLeft,
+            child: Text(
+              S.of(context).pageDriveChatRequestsHeadline,
+              style: Theme.of(context).textTheme.titleLarge,
+            ),
+          ),
+          const SizedBox(height: 10.0),
+        ];
+        pendingRidesColumn.addAll(_pendingRidesList(pendingRides));
```
Comment (fynsta) [inline lib/drives/pages/drive_detail_page.dart:180]: you don't need `addAll` here, you could put  `... _pendingRidesList(pendingRides)` in the last line of the column array definition

File: lib/drives/pages/drive_detail_page.dart (hunk 1 above)
Comment (istzen) [reply lib/drives/pages/drive_detail_page.dart:375]: yeah this method right now is just a placeholder the real method will be provided when ich finish the chat. I'll leave it for now if that's okay with you

File: lib/drives/pages/drive_detail_page.dart (hunk 2 above)
Comment (istzen) [reply lib/drives/pages/drive_detail_page.dart]: thanks for the hint, and sorry that this took two commits.

File: lib/drives/pages/drive_detail_page.dart (hunk 1 above)
Comment (fynsta) [reply lib/drives/pages/drive_detail_page.dart:375]: sure but the placeholder can still be accurate -> i.e. it cannot return null so the signature is wrong

Comment (istzen) [reply lib/drives/pages/drive_detail_page.dart]: yes that right i changed it

**Review by fynsta — APPROVED** [review]

**Review by Eddie-42 — APPROVED** [review]

**Review by MarkFalke — APPROVED** [review]

**Comment by fynsta** [comment]
@istzen So did you want to remove the pendingRides stuff that's still here on this branch? In the chat page

**Review by Showcas — APPROVED** [review]

**Comment by istzen** [comment]
> @istzen So did you want to remove the pendingRides stuff that's still here on this branch? In the chat page

yeah i just did it in the chat branch

**Pull request merged**